src/Models/Services/LocalStorageManager/LocalStorageManager.ts

```diff
@@ -0,0 +1,606 @@
+                    : null;
+
+                setSelectedAdtInstanceInLocalStorage(oldInstanceUrl);
+                localStorage.removeItem(SelectedEnvironmentLocalStorageKey);
+
+                return getEnvironmentItemFromResource(
+                    oldInstanceUrl,
+                    AzureResourceTypes.DigitalTwinInstance
+                );
+            }
+            // END of migration
+        }
+        return environmentConfigurationInLocalStorage?.selectedAdtInstance;
+    } catch (error) {
+        console.error(error.message);
+        return null;
+    }
+};
+
+/** To get the selected Storage account information from local storage
+ * @returns selected Storage account as an environment item from local storage, null if not exists
+ */
+export const getSelectedStorageAccountFromLocalStorage = (): EnvironmentItemInLocalStorage | null => {
+    const environmentConfigurationInLocalStorage = getEnvironmentConfigurationFromLocalStorage();
+    return environmentConfigurationInLocalStorage?.selectedStorageAccount;
+};
+
+/** To get the selected Storage container information from local storage
+ * @returns selected Storage container as an environment item from local storage, null if not exists
+ */
+export const getSelectedStorageContainerFromLocalStorage = (): EnvironmentItemInLocalStorage | null => {
+    try {
+        const environmentConfigurationInLocalStorage = getEnvironmentConfigurationFromLocalStorage();
+        if (!environmentConfigurationInLocalStorage?.selectedStorageContainer) {
+            // START of migration of values using old local storage key
+            const previouslyUsedKey = SelectedContainerLocalStorageKey;
+            const oldContainerUrl = localStorage.getItem(previouslyUsedKey);
+            if (oldContainerUrl) {
+                const storageAccountUrl = getStorageAccountUrlFromContainerUrl(
+                    oldContainerUrl
+                );
+
+                setSelectedStorageContainerInLocalStorage(
+                    getContainerNameFromUrl(oldContainerUrl),
+                    storageAccountUrl
+                );
+                setSelectedStorageAccountInLocalStorage(storageAccountUrl);
+                localStorage.removeItem(previouslyUsedKey);
+
+                return getEnvironmentItemFromResource(
+                    getContainerNameFromUrl(oldContainerUrl),
+                    AzureResourceTypes.StorageBlobContainer,
+                    storageAccountUrl
+                );
+            }
+        }
+        // END of migration
+        return environmentConfigurationInLocalStorage?.selectedStorageContainer;
+    } catch (error) {
+        console.error(error.message);
+        return null;
+    }
+};
+
+/** Updates arbitrary key-value pair in the local storage
+ * @param key key of the item in localstorage
+ * @param value value of the item in localstorage
+ */
+export const setLocalStorageItem = (key: string, value: string): void => {
+    if (key) {
```

Comment:
Should this log an error in case there is no key present?

Reply:
added